index.html

```diff
@@ -28,72 +30,19 @@
     </nav>
 
     <div id="shapes-background">
-      <img
-        class="rectangle-4"
-        src="./assets/images/Rectangle 4.svg"
-        alt="Rectangle 4"
-      />
-      <img
-        class="ellipse-3"
-        src="./assets/images/Ellipse 3.svg"
-        alt="Ellipse 3"
-      />
-      <img
-        class="ellipse-8"
-        src="./assets/images/Ellipse 8.svg"
-        alt="Ellipse 8"
-      />
-      <img
-        class="rectangle-6"
-        src="./assets/images/Rectangle 6.svg"
-        alt="Rectangle 6"
-      />
-      <img
-        class="ellipse-11"
-        src="./assets/images/Ellipse 11.svg"
-        alt="Ellipse 11"
-      />
-      <img
-        class="rectangle-7"
-        src="./assets/images/Rectangle 7.svg"
-        alt="Rectangle 7"
-      />
-      <img
-        class="ellipse-5"
-        src="./assets/images/Ellipse 5.svg"
-        alt="Ellipse 5"
-      />
-      <img
-        class="ellipse-14"
-        src="./assets/images/Ellipse 14.svg"
-        alt="Ellipse 14"
-      />
-      <img
-        class="ellipse-12"
-        src="./assets/images/Ellipse 12.svg"
-        alt="Ellipse 12"
-      />
-      <img
-        class="ellipse-13"
-        src="./assets/images/Ellipse 13.svg"
-        alt="Ellipse 13"
-      />
-      <img
-        class="ellipse-15"
-        src="./assets/images/Ellipse 15.svg"
-        alt="Ellipse 15"
-      />
-      <img
-        class="rectangle-5"
-        src="./assets/images/Rectangle 5.svg"
-        alt="Rectangle 5"
-      />
-      <img
-        class="ellipse-4"
-        src="./assets/images/Ellipse 4.svg"
-        alt="Ellipse 4"
-      />
-      
+      <img class="rectangle-4" src="./assets/images/Rectangle 4.svg" alt="Rectangle 4" />
+      <img class="ellipse-3" src="./assets/images/Ellipse 3.svg" alt="Ellipse 3" />
+      <img class="ellipse-8" src="./assets/images/Ellipse 8.svg" alt="Ellipse 8" />
+      <img class="rectangle-6" src="./assets/images/Rectangle 6.svg" alt="Rectangle 6" />
+      <img class="ellipse-11" src="./assets/images/Ellipse 11.svg" alt="Ellipse 11" />
+      <img class="rectangle-7" src="./assets/images/Rectangle 7.svg" alt="Rectangle 7" />
+      <img class="ellipse-5" src="./assets/images/Ellipse 5.svg" alt="Ellipse 5" />
+      <img class="ellipse-14" src="./assets/images/Ellipse 14.svg" alt="Ellipse 14" />
+      <img class="ellipse-12" src="./assets/images/Ellipse 12.svg" alt="Ellipse 12" />
+      <img class="ellipse-13" src="./assets/images/Ellipse 13.svg" alt="Ellipse 13" />
+      <img class="ellipse-15" src="./assets/images/Ellipse 15.svg" alt="Ellipse 15" />
+      <img class="rectangle-5" src="./assets/images/Rectangle 5.svg" alt="Rectangle 5" />
+      <img class="ellipse-4" src="./assets/images/Ellipse 4.svg" alt="Ellipse 4" />
```

Comment:
Please notice that these images (inside "shapes-background") shouldn't be an `img` element. Instead, they should be the [background image](https://www.w3schools.com/cssref/pr_background-image.php) of your container. This will make your code semantically accurate and give you an easier time with styling 😃. You can [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information.

Reply:
Thanks for the advice! 😁